datastore-api/app/routers/kgs.py

```diff
@@ -475,6 +476,81 @@ async def get_edges_by_id_as_nodes(
     else:
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Could not find edge.")
 
+
+
+@router.get(
+    "/{kg_name}/edges/extra_nodes",
+    summary="Get list of nodes which go in or out of given node for given knowledge graph",
+    description="Returns a list of nodes, which either go in or out of the given node.",
+    responses={
+        200: {
+            "description": "List of nodes",
+        },
+        404: {"description": "The edges could not be retrieved"},
+        500: {"model": HTTPError, "description": "Model API error"},
+    },
+)
+async def get_edges_by_id_as_nodes(
+    kg_name: str = Path(..., description="The name of the edge"),
+    docid: List = Body(..., description="The name of the edge to retrieve"),
```

Comment:
Seems like a wrong variable name. And please add more details about the type, `List[str]`, I guess